erpnext/stock/doctype/purchase_receipt/purchase_receipt.py

```
@@ -1521,6 +1526,35 @@ def set_missing_values(source, target):
 		target.purpose = "Material Transfer"
 		target.set_missing_values()
 
+	def update_item(source_doc, target_doc, source_parent):
```

Ruff (0.14.8)

Unused function argument: `source_parent`

(ARG001)